extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java

```diff
@@ -183,6 +185,19 @@ protected ModeledConnection acquire(RemoteAuthenticatedUser user,
             @Override
             public int compare(ModeledConnection a, ModeledConnection b) {
 
+                Directory<ConnectionGroup> directory;
+                ConnectionGroup group;
+				try {
```

Comment:
Please do not indent with tabs - our code uses a 4-space indent absolutely everywhere. Deviating from this causes horrible readability issues like:

![please-no-tabs-thanks](https://cloud.githubusercontent.com/assets/4632905/18239246/da8386f4-72f8-11e6-94a3-30ab1964e09f.png)

You may need to change your editor's settings. Even after that, it's worth double-checking the diff to make sure something crazy like the above hasn't happened.